File: 1-easy/problems/3340-check-balanced-string/solution.ts
function isBalanced(num: any): boolean | number | string | any {
  """
  :type num: str
  :rtype: bool
  """
  return sum(ord(num[i])-ord('0') for i in Array.from({length: 0, (num}, (_, i) => i).length, 2)) == sum(ord(num[i])-ord('0') for i in Array.from({length: 1, (num}, (_, i) => i).length, 2));
}

export { isBalanced };

File: 1-easy/problems/3340-check-balanced-string/solution.py
class Solution:
    def isBalanced(self, num):
        """
        :type num: str
        :rtype: bool
        """
        return sum(ord(num[i])-ord('0') for i in range(0, len(num), 2)) == sum(ord(num[i])-ord('0') for i in range(1, len(num), 2))
